server/src/test/java/org/elasticsearch/action/admin/cluster/storedscripts/PutStoredScriptRequestTests.java

```diff
@@ -48,4 +51,30 @@ public void testSerialization() throws IOException {
             }
         }
     }
+
+    public void testToXContent() throws IOException {
```

Comment:
it would be reasonable `PutStoredScriptRequestTests` to extend `AbstractStreamableXContentTestCase<PutStoredScriptRequest>` - it gives more randomization and general test flow for requests/responses - like you do for `PutStoredScriptResponseTests`

Reply:
Thanks for your comment. But I don't know the comment for ping reviewers. Is there any documentation about this? Or could you help me to assign reviewer for the review again... Thanks!

Reply:
I meant to leave a comment like `hey @reviewer - could you pls have a look`